.coveragerc

```diff
@@ -1,6 +1,6 @@
 [run]
 branch=True
-source=minion
+source=minion, common
```

Comment:
Now that you already have a real Django app, do you still need to have the `minion` app? I'd recommend that you remove it.